assets/src/blocks/TableOfContents/TableOfContentsItems.js

```diff
@@ -21,8 +21,8 @@ export const SubmenuItems = ({menuItems}) => {
   };
 
   return menuItems.length > 0 && (
-    <div className="submenu-menu">
-      <ul className="submenu-item">
+    <div className="table-of-contents-menu">
```

Comment:
I'm wondering if this could be an issue for existing blocks, that the old CSS classes no longer work 🤔 But [it seems not](https://www-dev.greenpeace.org/test-venus/privacy-and-cookies/)

Reply:
I couldn't find any visual differences, either.